File: CS/CustomDrawCrosshairSample/Model/NwindDbContext.cs
namespace CustomDrawCrosshairSample.Model {
    using System;
    using System.Data.Entity;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Linq;

    public partial class NwindDbContext : DbContext {
        public NwindDbContext()
            : base("name=NwindDbContext") {
        }

        public virtual DbSet<Employee> Employees { get; set; }
        public virtual DbSet<Order> Orders { get; set; }

        protected override void OnModelCreating(DbModelBuilder modelBuilder) {

        }
    }
}

File: VB/CustomDrawCrosshairSample/Model/NwindDbContext.vb
Imports System.Data.Entity
Imports System.ComponentModel.DataAnnotations.Schema

Namespace CustomDrawCrosshairSample.Model

    Public Partial Class NwindDbContext
        Inherits DbContext

        Public Sub New()
            MyBase.New("name=NwindDbContext")
        End Sub

        Public Overridable Property Employees As DbSet(Of Employee)

        Public Overridable Property Orders As DbSet(Of Order)

        Protected Overrides Sub OnModelCreating(ByVal modelBuilder As DbModelBuilder)
        End Sub
    End Class
End Namespace
